pkg/types/layer.go

```diff
@@ -502,7 +543,7 @@ func getImportFromInterface(v interface{}) (Import, error) {
 	// if it's not a map then it's a string
 	s, ok := v.(string)
 	if ok {
-		return Import{Hash: "", Path: fmt.Sprintf("%v", s)}, nil
+		return Import{Hash: "", Path: fmt.Sprintf("%v", s), Dest: "", Uid: uid, Gid: gid}, nil
```

Comment:
This is a bugfix for the import/copy pr?

Reply:
Definitely a bug, since the "zero" value doesn't set other fields correctly.

Reply:
Ok - ideally that would be a separate 2-line commit (in part to make later cherrypicks and reverts easier).